node/src/BaseClient.ts

```diff
@@ -2827,6 +2828,85 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Returns a random element from the sorted set stored at `key`.
+     *
+     * See https://valkey.io/commands/zrandmember/ for more details.
+     *
+     * @param keys - The key of the sorted set.
+     * @returns A string representing a random element from the sorted set.
+     *     If the sorted set does not exist or is empty, the response will be `null`.
+     *
+     * @example
+     * ```typescript
+     * const payload1 = await client.zrandmember("mySortedSet");
+     * console.log(payload1); // Output: "Glide" (a random member from the set)
+     *
+     * const payload2 = await client.zrandmember("nonExistingSortedSet");
+     * console.log(payload2); // Output: null
+     * ```
+     */
+    public async zrandmember(key: string): Promise<string | null> {
```

Comment:
consider: 
```suggestion
    public async zrandmember(key: string, count?: number): Promise<string | null | string[]> {
```

Reply:
let's not do this.  just because we can merge signatures, it's not always the right thing to do.  
it would reduce the number of commands we need to maintain, but it means we have to rely on the user reading and understanding the documentation